gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx

```diff
@@ -17,56 +17,47 @@ screenshot: /docs/projects/images/python.png
 maturity: Released
 ---
 
-(Version 2.4.1)
-
-Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on [matrix-nio](https://github.com/poljar/matrix-nio).
+Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on [matrix-nio](https://github.com/poljar/matrix-nio)
```

Comment:
Changes to this file look like they have nothing to do with the topic of this PR. Did you really mean to include that?

Reply:
🤔 Good spot. I'm not sure what happened here. I have undone these changes.